core/controller/src/main/scala/whisk/core/controller/Triggers.scala

```diff
@@ -300,13 +305,21 @@ trait WhiskTriggersApi extends WhiskCollectionAPI {
   }
 
   /**
-   * Iterates through each active rule and invoke each mapped action.
+   * Iterates through each rule and invoking each active rule's mapped action.
    */
   private def activateRules(user: Identity,
                             args: JsObject,
                             rulesToActivate: Map[FullyQualifiedEntityName, ReducedRule])(
     implicit transid: TransactionId): Iterable[Future[JsObject]] = {
     rulesToActivate.map {
+      case (ruleName, rule) if (rule.status != Status.ACTIVE) =>
+        Future {
```

Comment:
Please use `Future.successful`. Using `Future.apply` (as done here) results in a forced async-dispatch which is not needed here.